kratos/geometries/line_2d_2.h

```diff
@@ -641,13 +644,28 @@ class Line2D2 : public Geometry<TPointType>
         return 2;
     }
 
+    /**
+     * @brief This method gives you all edges of this geometry.
+     * @details This method will gives you all the edges with one dimension less than this geometry.
+     * @return GeometriesArrayType containes this geometry edges.
+     * @see EdgesNumber()
+     * @see Edge()
+     */
+    GeometriesArrayType Edges( void ) override
+    {
+        GeometriesArrayType edges = GeometriesArrayType();
+        edges.push_back( Kratos::make_shared<EdgeType>( this->pGetPoint( 0 ), this->pGetPoint( 1 ) ) );
+        edges.push_back( Kratos::make_shared<EdgeType>( this->pGetPoint( 1 ), this->pGetPoint( 0 ) ) );
```

Comment:
why returning two edges?

Reply:
Because in the current implementation the number of egdes is 2: https://github.com/KratosMultiphysics/Kratos/blob/58630976bb9945dd1fcb548adf1ef8f3060c8a57/kratos/geometries/line_2d_2.h#L639

Not my chose